src/pages/components/VideoCard.tsx

```diff
@@ -10,31 +21,35 @@ export function VideoCard() {
                     Ordino is a browser-based visual data analysis solution to flexibly rank, filter, and explore genes,
                     cell lines, and tissue samples based on a rich set of experimental and metadata.
         </Card.Text>
-                <Card.Title>
+                <Card.Title >
                     <i className="mr-2 fas fa-check"></i>Features
-        </Card.Title>
+                </Card.Title>
                 <Card.Text>
-                    You can find an overview of the main features here.
-        </Card.Text>
+                    You can find an overview of the main features
+                    <Link {...newWindowProps} className="card-link" to="/features">&nbsp;here.</Link>
```

Comment:
Is the class `card-link` on all the links necessary? I could not find any styles or selector for it.

Reply:
It is a bootstrap class https://getbootstrap.com/docs/4.0/components/card/.
I prefer it to the react router default link style. But you may remove it if you like.

Reply:
Ok, thanks for the explanation. Then I would keep the class.